core/primitives/src/version.rs

```diff
@@ -59,21 +59,31 @@ pub const FIXED_MINIMUM_NEW_RECEIPT_GAS_VERSION: ProtocolVersion = 66;
 /// it’s set according to the schedule for that protocol upgrade.  Release
 /// candidates usually have separate schedule to final releases.
 pub const PROTOCOL_UPGRADE_SCHEDULE: Lazy<ProtocolUpgradeVotingSchedule> = Lazy::new(|| {
-    // Update to according to schedule when making a release.
-    // Keep in mind that the protocol upgrade will happen 1-2 epochs (15h-30h)
-    // after the set date. Ideally that should be during working hours.
-    // e.g. ProtocolUpgradeVotingSchedule::from_env_or_str("2000-01-01 15:00:00").unwrap());
-
-    ProtocolUpgradeVotingSchedule::default()
+    // Update according to the schedule when making a release. Keep in mind that
+    // the protocol upgrade will happen 1-2 epochs (15h-30h) after the set date.
+    // Ideally that should be during working hours.
+    //
+    // Multiple protocol version upgrades can be scheduled. Please make sure
+    // that they are ordered by datetime and version, the last one is the
+    // PROTOCOL_VERSION and that there is enough time between subsequent
+    // upgrades.
+
+    // e.g.
+    // let v1_protocol_version = 50;
+    // let v2_protocol_version = 51;
+    // let v1_datetime = ProtocolUpgradeVotingSchedule::parse_datetime("2000-01-01 15:00:00").unwrap();
+    // let v2_datetime = ProtocolUpgradeVotingSchedule::parse_datetime("2000-01-10 15:00:00").unwrap();
+    //
+    // let schedule = vec![(v1_datetime, v1_protocol_version), (v2_datetime, v2_protocol_version)];
+    // ProtocolUpgradeVotingSchedule::new_from_env_or_schedule(PROTOCOL_VERSION, schedule).unwrap()
+
+    ProtocolUpgradeVotingSchedule::new_from_env_or_schedule(PROTOCOL_VERSION, vec![]).unwrap()
```

Comment:
hm why undo the `new_immediate()` call here in 4c66cbd8fa1b97f45ed8f116f316c18480e6caa5 ? I kinda liked it that way better, but up to you

Reply:
I need the env override for some of the tests to work.